### PR TITLE
[HS-171] go to settings

### PR DESCRIPTION
**Pull request by hophongit98**

### CODE REVIEW

File: lib/features/add_property/view/widgets/add_property_images_view.dart [hunk 1]
```diff
@@ -13,28 +16,62 @@ class AddPropertyImagesView extends StatelessWidget {
...
     return BlocListener<AddPropertyImagesCubit, AddPropertyImagesState>(
-      listener: (context, state) {
+      listener: (_, state) {
+        if (state is PhotoPermissionGranted) {
+          _showSelectPhotoBottomSheet(context).then((isShown) {
+            context
+                .read<AddPropertyImagesCubit>()
+                .dismissSelectPhotoPermissionBottomSheet(isShown);
+          });
+          context.read<AddPropertyImagesCubit>().openSelectPhotoBottomSheet();
```
Comment (suesitran) [inline lib/features/add_property/view/widgets/add_property_images_view.dart:62]: leave this to the Select photo bottom sheet, this event will be fired by Select photo bottom sheet

File: lib/features/add_property/view/widgets/add_property_images_view.dart [hunk 2]
```diff
@@ -13,28 +16,62 @@ class AddPropertyImagesView extends StatelessWidget {
...
   @override
   Widget build(BuildContext context) {
     return BlocListener<AddPropertyImagesCubit, AddPropertyImagesState>(
-      listener: (context, state) {
+      listener: (_, state) {
+        if (state is PhotoPermissionGranted) {
+          _showSelectPhotoBottomSheet(context).then((isShown) {
+            context
+                .read<AddPropertyImagesCubit>()
+                .dismissSelectPhotoPermissionBottomSheet(isShown);
```
Comment (suesitran) [inline lib/features/add_property/view/widgets/add_property_images_view.dart:60]: change name: onSelectPhotoBottomSheetDismissed()

File: lib/features/add_property/view/widgets/add_property_images_view.dart [hunk 3]
```diff
@@ -13,28 +16,62 @@ class AddPropertyImagesView extends StatelessWidget {
...
         if (state is PhotoPermissionDeniedForever) {
-          // open app setting
-        }
-
-        if (state is PhotoPermissionGranted) {
-          // open photo screen
+          _showGoToSettingBottomSheet(context).then((isShown) {
+            context
+                .read<AddPropertyImagesCubit>()
+                .dismissPhotoPermissionBottomSheet(isShown);
```
Comment (suesitran) [inline lib/features/add_property/view/widgets/add_property_images_view.dart:73]: change name: onPhotoPermissionBottomSheetDismissed()

**Review by suesitran — COMMENTED** [review]
some comments

File: lib/features/add_property/view/widgets/add_property_images_view.dart (hunk 1 above)
Comment (hophongit98) [reply lib/features/add_property/view/widgets/add_property_images_view.dart]: already removed

File: lib/features/add_property/view/widgets/add_property_images_view.dart (hunk 2 above)
Comment (hophongit98) [reply lib/features/add_property/view/widgets/add_property_images_view.dart]: renamed

File: lib/features/add_property/view/widgets/add_property_images_view.dart (hunk 3 above)
Comment (hophongit98) [reply lib/features/add_property/view/widgets/add_property_images_view.dart]: renamed

**Review by suesitran — APPROVED** [review]

**Pull request merged**